TABTestKit/Classes/Contexts/KeyboardContext.swift

```diff
@@ -11,7 +11,7 @@ import XCTest
 public protocol KeyboardContext {}
 public extension KeyboardContext {
 	
-	func keyboard(isType type: Keyboard.KeyboardType) {
+	func keyboardIsType(_ type: Keyboard.KeyboardType) {
```

Comment:
What about if we just renamed the global `keyboard` instance instead? To something like `softwareKeyboard` or something?

Reply:
Can do if you feel that's the best way to go. Just seems odd to me to me to make this  instance name more specific when it's a global instance and and we don't gave more than one (as the docs state). It might then lead you to think there's a `hardwareKeyboard` instance somewhere else within TTK. Minor thing but you'd also have a misalignment with the instance name and the type name too so this just felt like the smaller change.

Reply:
I don't mind really, I just hate the non-Swifty function signature 😭

Reply:
Some alternatives: 

1. `func typeOfKeyboard(is type: Keyboard.KeyboardType)`
2. `func keyboardType(is type: Keyboard.KeyboardType)`

Reply:
I like number 2, will make changes now 👍

Reply:
Nice👌

Reply:
@KaneCheshire Are you happy with option 2?

Reply:
Yep :)